Add Cryptorank envs (#310)
blockscout/docs: setup-and-run-blockscout/env-variables/backend-env-variables.md

| `EXCHANGE_RATES_CRYPTORANK_SECONDARY_COIN_ID`        | Sets Cryptorank coin ID for secondary coin market chart. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: (empty)<br>Applications: API, Indexer</p>            |
| `EXCHANGE_RATES_CRYPTORANK_PLATFORM_ID`              | Sets Cryptorank platform ID. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: (empty)<br>Applications: API, Indexer</p>            |

> | `EXCHANGE_RATES_CRYPTORANK_BASE_URL`                 | If set, overrides the Cryptorank API url. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: <code>coin_gecko</code><br>Applications: API, Indexer</p>            |

Reviewer: > <code>coin_gecko</code>

I guess there should be `https://api.cryptorank.io/v1/`

Reviewer: Also check the entry for `EXCHANGE_RATES_CRYPTORANK_API_KEY`

| `EXCHANGE_RATES_CRYPTORANK_API_KEY`                  | Cryptorank API key. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: <code>https://api.cryptorank.io/v1/</code><br>Applications: API, Indexer</p>            |
| `EXCHANGE_RATES_CRYPTORANK_COIN_ID`                  | Sets Cryptorank coin ID. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: (empty)<br>Applications: API, Indexer</p>            |

> | `EXCHANGE_RATES_CRYPTORANK_LIMIT`                    | Sets Cryptorank "batch size" (limit on amount of prices returned for one request) for for tokens price fetching. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: <code>1000</code><br>Applications: API, Indexer</p>            |

Reviewer: I wanted to notice repetition of _for_, but also refactored description. You may leave the one you wrote if this one seems to be less clear.

```suggestion
| `EXCHANGE_RATES_CRYPTORANK_LIMIT`                    |Sets the maximum number of token prices returned in a single request. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: <code>1000</code><br>Applications: API, Indexer</p>            |
```

Author: Nice catch! Fixed
